addons/hr_homeworking/models/hr_employee.py

```diff
@@ -19,6 +19,10 @@ class HrEmployeeBase(models.AbstractModel):
     friday_location_id = fields.Many2one('hr.work.location', string='Friday')
     saturday_location_id = fields.Many2one('hr.work.location', string='Saturday')
     sunday_location_id = fields.Many2one('hr.work.location', string='Sunday')
+    current_location_id = fields.Many2one(
```

Comment:
For the help.. bedo says:
_Do we need a help attribute rather than a comment given it's not appearing in any view?_

It does not appear.. but maybe my bad, but I use help sometimes to describe a bit of a field, if it's not supper straightforward .. I could have added comment instead, but I don't think help hurts